### ISSUE TITLE
Function

### ISSUE THREAD

**Issue by Sy1mple** [jaygshah/Robust-Brain-Age-Prediction#2]
![image](https://github.com/user-attachments/assets/4d068a9d-66b6-41c5-ae78-e279caf42b6f)
![image](https://github.com/user-attachments/assets/51d72cb2-f446-4b4b-9293-498cc75a4e56)

What is abut this?Which function is short for?

**Comment by jaygshah** [comment]
Can you share your training command? I was able to run using `python3 train.py --dataset data --batch_size 2 --losses ce order --model_name resnet18 --ld 0.1 `in my local just now.

**Comment by Sy1mple** [comment]
I use this but no reply and right-clicking to run the file train.py gives that error
![image](https://github.com/user-attachments/assets/4f87d1e3-f451-411a-802e-42c873086004)

**Comment by Sy1mple** [comment]
Soryy i got this issue

**Comment by jaygshah** [comment]
I am confused what your error is here. Is it still related to the loss functions' usage or something else, can you share that? In the last screenshot it seems to have worked. Also I see you're using the training_folder same as mine, which I believe might not work for you.
